ansible/inventory/group_vars/operator-pipeline.yml

```diff
@@ -5,6 +5,7 @@ oc_namespace: "operator-pipeline-{{ env }}"
 service_account_name: operator-pipeline-admin
 suffix: "{{ env }}"
 preflight_min_version: 0.0.0
+preflight_trigger_environment: preprod
```

Comment:
The variable should be also defined for the production environment.

Reply:
Done